Source/WebCore/PAL/pal/PALSwift/CryptoKitShim.swift

```diff
@@ -131,7 +134,8 @@ public final class AesKw: NSObject {
         do {
             let cipher = try AES.KeyWrap.wrap(
                 SymmetricKey(data: UnsafeBufferPointer(start: data, count: Int(dataSize))),
-                using: SymmetricKey(data: UnsafeBufferPointer(start: key, count: Int(keySize))))
+                using: SymmetricKey(
+                    data: UnsafeBufferPointer(start: key, count: Int(keySize))))
```

Comment:
Need a wrapper that expresses a safe way to construct a key.

Reply:
There are other cases like this too, but I won't comment on each one. One way to get a close approximation is to search for "unsafe" in the code; though some unsafe pointers will instantiate implicitly.

Reply:
https://github.com/WebKit/WebKit/pull/25602#discussion_r1531120724